src/Components/BoardGame/index.js

```diff
@@ -65,10 +65,15 @@ const BoardGame = () => {
     });
   };
 
+  const hoverEffect = () => {
+    document.documentElement.style.setProperty('--currentPlayer-color', opposingPlayer);
```

Comment:
can't you just add class `active` to whatever player and then handle everything in CSS?

Reply:
Most of it is taken care of in CSS. After each player's turn, I needed something to change the hover color. I chose to change the variable in CSS.

Reply:
imo it looks slightly broken from UX perspective. Only the top row has onHover effect, it should be applied on an entire column and the coin should appear either above the grid or it should appear in place of a possible move.

If it is too hard to implement then you can skip this functionality because it is not a main requirement